common/web/gesture-recognizer/src/test/resources/json/receiver/embeddedBorderCancel.json

```diff
@@ -143,12 +143,12 @@
               },
               {
                 "targetX": 118,
-                "targetY": 4,
+                "targetY": 5,
```

Comment:
The 'safe bound' applied during the check is ` < 4`.  Unfortunately, since there tend to be subpixel calculations when testing on real Android test devices, it's possible to get something akin to `3.9999` (rather than a properly-integer `4`) due to floating-point approximations, and that affects the check.

This correction provides an ample "fudge-factor" to buffer against such numerical approximation errors so that they don't adversely fail the corresponding automated test.